NickvisionTubeConverter.Shared/Helpers/DependencyManager.cs

```diff
@@ -14,21 +15,16 @@ internal static class DependencyManager
     public static string PythonPath
     {
         get
-        {
-            var prefixes = new List<string>() {
-                Directory.GetParent(Directory.GetParent(Path.GetFullPath(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!))!.FullName)!.FullName,
-                Directory.GetParent(Path.GetFullPath(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!))!.FullName,
-                "/usr"
-            };
-            foreach (var prefix in prefixes)
+        { 
+            foreach (var dir in SystemDirectories.Path)
             {
-                var path = $"{prefix}/bin/python3";
+                var path = $"{dir}{Path.DirectorySeparatorChar}python3";
                 if (File.Exists(path))
                 {
                     return path;
                 }
             }
-            return "python3";
+            throw new Exception("python3 not in path");
```

Comment:
I'd still `return "python3"` in case we bundle it with the app on windows let's say in the same directory it will just pull it from there. Same for other dependencies.

Or maybe:
```csharp
if(File.Exists("python3'))
{
    return "python3";
}
throw new Exception("python3 not in path");
```
*again same for the other dependencies*

Reply:
I doubt it's good idea to place all dependencies in the same directory. And we could modify PATH to point to all required directories before DependencyManager resolves all deps. So I think for now it's better to keep as it is instead of adding unused code that may be a subject to change.